File: backEnd/services/doctor-service/src/main/java/com/cloudbrainmed/doctor/entity/MedicalOrder.java
package com.cloudbrainmed.doctor.entity;

import java.time.LocalDateTime;

/**
 * 医技申请主表实体（映射 medical_order 表）
 * 统含检查(EXAM)和检验(LAB)两类申请
 */
public class MedicalOrder {
    private String orderId;
    private String patientId;
    private String registerId;
    private String doctorId;
    private String clinicalSummary;
    private String urgencyLevel;
    private String sourceType;
    private String aiTraceId;
    private String status;
    private String payStatus;
    private LocalDateTime confirmedTime;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;

    public String getOrderId() { return orderId; }
    public void setOrderId(String orderId) { this.orderId = orderId; }
    public String getPatientId() { return patientId; }
    public void setPatientId(String patientId) { this.patientId = patientId; }
    public String getRegisterId() { return registerId; }
    public void setRegisterId(String registerId) { this.registerId = registerId; }
    public String getDoctorId() { return doctorId; }
    public void setDoctorId(String doctorId) { this.doctorId = doctorId; }
    public String getClinicalSummary() { return clinicalSummary; }
    public void setClinicalSummary(String clinicalSummary) { this.clinicalSummary = clinicalSummary; }
    public String getUrgencyLevel() { return urgencyLevel; }
    public void setUrgencyLevel(String urgencyLevel) { this.urgencyLevel = urgencyLevel; }
    public String getSourceType() { return sourceType; }
    public void setSourceType(String sourceType) { this.sourceType = sourceType; }
    public String getAiTraceId() { return aiTraceId; }
    public void setAiTraceId(String aiTraceId) { this.aiTraceId = aiTraceId; }
    public String getStatus() { return status; }
    public void setStatus(String status) { this.status = status; }
    public String getPayStatus() { return payStatus; }
    public void setPayStatus(String payStatus) { this.payStatus = payStatus; }
    public LocalDateTime getConfirmedTime() { return confirmedTime; }
    public void setConfirmedTime(LocalDateTime confirmedTime) { this.confirmedTime = confirmedTime; }
    public LocalDateTime getCreateTime() { return createTime; }
    public void setCreateTime(LocalDateTime createTime) { this.createTime = createTime; }
    public LocalDateTime getUpdateTime() { return updateTime; }
    public void setUpdateTime(LocalDateTime updateTime) { this.updateTime = updateTime; }
}

File: FastApi-backend/exam-service/app/models/medical_order.py
from sqlalchemy import Column, String, DateTime, Text
from app.core.database import Base
from datetime import datetime


class MedicalOrder(Base):
    __tablename__ = "medical_order"

    order_id = Column(String(64), primary_key=True, nullable=False, comment="申请单ID")
    patient_id = Column(String(64), nullable=False, comment="患者ID")
    register_id = Column(String(64), nullable=False, comment="挂号ID")
    doctor_id = Column(String(64), nullable=False, comment="开单医生ID")
    clinical_summary = Column(Text, nullable=True, comment="临床摘要")
    urgency_level = Column(String(20), nullable=True, comment="紧急程度")
    source_type = Column(String(20), nullable=True, comment="来源类型：EXAM检查/LAB检验")
    ai_trace_id = Column(String(64), nullable=True, comment="AI追踪ID")
    status = Column(String(20), nullable=True, comment="状态")
    pay_status = Column(String(20), nullable=True, comment="支付状态")
    confirmed_time = Column(DateTime, nullable=True, comment="确认时间")
    create_time = Column(DateTime, default=datetime.now, comment="创建时间")
    update_time = Column(DateTime, default=datetime.now, onupdate=datetime.now, comment="更新时间")